rules-reviewed/eap7/eap4and5/jboss-eap4and5to6and7-xml.windup.xml

```diff
@@ -26,8 +26,8 @@
                 <hint issue-display-mode="all" title="Replace service-style deployments" effort="1" category-id="optional">
                     <message>
                         <![CDATA[
-                        MBeans were part of the core architecture in previous versions of Red Hat JBoss Enterprise Application Platform.  
```

Comment:
is this double space removal on purpose ? we put it to create a new line after the text.

Reply:
That was removed by my editor. It's set to remove trailing whitespace, so this was not intentional. I'll turn that feature off. 

At the same time, I think it would be safer to put a blank line to indicate a new line, but that's your call.

Reply:
I see your point and I agree it's clearer, but if I'm not wrong the effect is not the same.

Reply:
Yes, you are correct. I will revert those changes.